src/flexmeasures_client/client.py

```diff
@@ -85,6 +86,19 @@ def __post_init__(self):
             )
         if len(self.password) < 1:
             raise EmptyPasswordError("password cannot be empty")
+        self.determine_port()
+
+    def determine_port(self):
+        parts = self.host.split(':')
+        if len(parts) > 1:
+            if self.port is not None:
```

Comment:
Adding `and self.port != int(parts[1])` would prevent an error message like this:

```
flexmeasures_client.exceptions.WrongHostError: Cannot set port=5000 and also as part of host=127.0.0.1:5000
```

Reply:
To me, it makes more sense to raise the error also in this case, as it is redundant and likely a mistake to pass it both ways.